docs/scos/dev/front-end-development/202212.0/oryx/reactivity/signals.md

```diff
@@ -0,0 +1,124 @@
+---
+title: [Add title; the file name and title must be identic]
+description: [Add description]
+template: concept-topic-template
+last_updated: Jul 11, 2023
+---
+
+
+Signals offer a clean and efficient reactivity API for components in the Oryx framework. They allow for the declarative formulation of component logic while seamlessly integrating with observables from domain services.
+
+## Implementation of signals in Oryx
+
+The Oryx implementation of signals has a core mechanism and a simplified API. The core is well-suited for advanced usage, while the simplified API is sufficiently robust for most components. This document focuses on the simplified API.
+
+### Creating signals
+
+To create a signal, use the `signal()` function. This function can either take a raw value or accept an observable.
+
+Here's an example of creating a simple signal:
+
+```ts
+const counter = signal(1);
+```
+
+Changing the signal value:
+
+```ts
+counter.set(2);
+```
+
+Creating a signal from an observable:
+
+```ts
+const values = signal(observable$);
+```
+
+You can initialize signals with options to adjust their behavior:
+
+- `equal`: allows for a custom equality function between two consecutive signal values. By default, strict comparison is used. Implementing your own function gives control over when a signal updates, avoiding unnecessary updates and performance costs when new and old values are practically identical.
```

Comment:
`gives control over when a signal updates` Please clarify the meaning of this or update this sentence to improve clarity

Reply:
I've updated the docs with a slightly different intro to the signal topic and a small tweak in the text above.